go/test/endtoend/cluster/vtgate_process.go

```diff
@@ -226,3 +227,21 @@ func VtgateProcessInstance(port int, grpcPort int, mySQLServerPort int, cell str
 
 	return vtgate
 }
+
+// GetVars returns map of vars
+func (vtgate *VtgateProcess) GetVars() map[string]interface{} {
+	resultMap := make(map[string]interface{})
+	resp, err := http.Get(vtgate.VerifyURL)
+	if err != nil {
+		return nil
+	}
+	if resp.StatusCode == 200 {
+		respByte, _ := ioutil.ReadAll(resp.Body)
+		err := json.Unmarshal(respByte, &resultMap)
+		if err != nil {
+			panic(err)
```

Comment:
Let's not do panic,  log error and return nil

Reply:
Done. Refer PR46